bundles/org.openhab.binding.hive/src/main/java/org/openhab/binding/hive/internal/client/feature/TemperatureSensorFeature.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hive.internal.client.feature;
+
+import java.util.Objects;
+
+import javax.measure.Quantity;
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.hive.internal.client.BuilderUtil;
+import org.openhab.binding.hive.internal.client.FeatureAttribute;
+
+/**
+ *
+ *
+ * @author Ross Brown - Initial contribution
+ */
+@NonNullByDefault
+public final class TemperatureSensorFeature implements Feature {
+    private final FeatureAttribute<Quantity<Temperature>> temperature;
+
+    private TemperatureSensorFeature(
+            final FeatureAttribute<Quantity<Temperature>> temperature
+    ) {
+        Objects.requireNonNull(temperature);
+
+        this.temperature = temperature;
+    }
+
+    public FeatureAttribute<Quantity<Temperature>> getTemperature() {
+        return this.temperature;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static final class Builder {
```

Comment:
I don't think this class needs a builder for a single field

Reply:
I did this for consistency with the other `XxxFeature` classes.
It's overkill for a feature with a single attribute but I thought it was nicer from a library user perspective that you know to always use `XxxFeature.build()` instead of having to check which ones use a public constructor.